smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/serde/member/MemberShapeDecodeGenerator.kt

```diff
@@ -118,10 +119,11 @@ open class MemberShapeDecodeGenerator(
         val memberTimestampFormatTrait = memberShape.getTrait<TimestampFormatTrait>()
         val swiftTimestampFormatCase = TimestampUtils.timestampFormat(ctx, memberTimestampFormatTrait, timestampShape)
         return writer.format(
-            "try \$L.\$L(format: \$L)",
+            "try \$L.\$L(format: \$N\$L)",
```

Comment:
The `TimestampFormat` type is explicitly referenced so it gets the needed SPI import.

The same thing happens below where a timestamp format is rendered.